### ISSUE TITLE
Make an official release after #70 and #71

### ISSUE THREAD

**Issue by mengxr** [typesafehub/genjavadoc#73]
It would be nice to make an official release after #70 and #71 were merged, so we can switch to the official jars in Spark 2.0, which is coming soon. There might be other projects that need this feature too.

cc: @jodersky @rkuhn

**Comment by jodersky** [comment]
+1
Scala 2.12.0-M4 was just published, maybe these issues can be combined with an M4 genjavadoc release

**Comment by ktoso** [comment]
Yeah, good idea, let's put some on a milestone

**Comment by jodersky** [comment]
@2m, thanks for merging the PRs :)
Do you already know when you plan to make a next release? Spark 2.0 is slowly showing up on the horizon and it would be great to switch to the official genjavadoc before code freeze.

**Comment by ktoso** [comment]
Let's make sure it gets out in time for the 2.0, let us know if you have more info when that might be ;)

**Comment by mengxr** [comment]
cc @rxin, who manages Spark 2.0 release.

**Comment by jodersky** [comment]
From what I gather on the mailing list, it seems the freeze will be in 2 weeks. It would be awesome if we could get a release by then!

**Comment by 2m** [comment]
Alright. There has been lots of changes here since the last release. I will do a smoke test to see how the latest version works with current Akka and cut a release if everything looks good.

**Comment by mengxr** [comment]
@jodersky Could you test Spark with the latest genjavadoc master and see whether it works as expected? Thanks!

**Comment by mengxr** [comment]
FYI, I tested genjavadoc@g9b19d03 with Spark master ([diff](https://github.com/apache/spark/compare/master...mengxr:SPARK-14511?expand=1)) and got the following error:

~~~
[info] Loading source file /Users/meng/src/spark/core/target/java/org/apache/spark/util/TimeStampedHashMap.java...
[error] /Users/meng/src/spark/core/target/java/org/apache/spark/util/TimeStampedHashMap.java:132: error: <identifier> expected
[error]   static public  B default (A key)  { throw new RuntimeException(); }
[error]                   ^
[error] /Users/meng/src/spark/core/target/java/org/apache/spark/util/TimeStampedHashMap.java:132: error: illegal start of type
[error]   static public  B default (A key)  { throw new RuntimeException(); }
[error]                            ^
[error] /Users/meng/src/spark/core/target/java/org/apache/spark/util/TimeStampedHashMap.java:132: error: ';' expected
[error]   static public  B default (A key)  { throw new RuntimeException(); }
[error]                              ^
[error] /Users/meng/src/spark/core/target/java/org/apache/spark/util/TimeStampedHashMap.java:132: error: <identifier> expected
[error]   static public  B default (A key)  { throw new RuntimeException(); }
[error]                                  ^
~~~

**Comment by mengxr** [comment]
It seems that it forgot to escape `default`, which is a Java keyword.

**Comment by jodersky** [comment]
maybe that's related to #81 ? it seems there are some old issues with escaping keywords

**Comment by 2m** [comment]
I have released `0.10` for all of the scala versions defined in the build definition. It is on the way to maven central.

**Comment by jodersky** [comment]
Thanks!
